### PR TITLE
JOSS paper setup

### PR DESCRIPTION
**Pull request by gomezzz**
Adding initial files for the JOSS submission

### CODE REVIEW

**Comment by gomezzz** [comment]
@schuhmaj can we merge this? :) #9 builds on it, I think

File: paper/paper.bib [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+@inproceedings{GeodesyNetsBenchmark,
+  title={Investigation of the Robustness of Neural Density Fields},
+  author={Jonas Schuhmacher and Fabio Gratl and Dario Izzo and Pablo G{\'o}mez},
+  booktitle={Proceedings of the 9th International Conference on Astrodynamics Tools and Techniques, ICATT},
+  year={2023},
+}
+
+@inproceedings{marak2023trajectory,
```
Comment (schuhmaj) [inline paper/paper.bib:41]: Isn't it the other way around?
Rasmus' paper in
Proceedings of the 9th International Conference on Astrodynamics Tools and Techniques, ICATT
And mine in
Proceedings of the 12th International Conference on Guidance, Navigation \& Control Systems (GNC)

**Review by schuhmaj — CHANGES_REQUESTED** [review]
Is the citation correct? Otherwise fine. 

Anyway, Many thanks for setting this up 👍😀

**Review by schuhmaj — APPROVED** [review]
All good.

**Pull request merged**